Fabric.Authorization.Persistence.SqlServer/Stores/SqlServerRoleStore.cs

```diff
@@ -138,6 +139,10 @@ private IEnumerable<EntityModels.Role> GetRoleEntityModels(string grain, string
         {
             var roles = _authorizationDbContext.Roles
                 .Include(r => r.SecurableItem)
```

Comment:
Do we need to exclude deleted related entities here?

Reply:
Yeah - i'll fix that in a later PR.